cognite_toolkit/_cdf_tk/utils/interactive_select.py

```diff
@@ -115,6 +123,17 @@ def _aggregate_count(self, hierarchies: list[str], data_sets: list[str]) -> int:
         )
 
 
+class FileMetadataInteractiveSelect(AssetCentricInteractiveSelect):
+    def _aggregate_count(self, hierarchies: list[str], data_sets: list[str]) -> int:
+        result = self.client.files.aggregate(
+            filter=FileMetadataFilter(
+                data_set_ids=[{"externalId": item} for item in data_sets] or None,
+                asset_subtree_ids=[{"externalId": item} for item in hierarchies] or None,
+            )
+        )
+        return result[0].count if result else 0
```

Comment:
What's the type for "result" here? Is it FileMetadataList from the SDK?
Wondering what will happen if the indexed result has no "count" field.